packages/lib-react-components/src/FavouritesButton/FavouritesButton.spec.js

```diff
@@ -1,100 +1,76 @@
-import { shallow } from 'enzyme'
-import sinon from 'sinon'
-import MetaToolsButton from '../MetaToolsButton'
-
-import { FavouritesButton } from './FavouritesButton'
-import HeartIcon from './HeartIcon'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { composeStory } from '@storybook/react'
+import Meta, { NotFavourited, Favourited, Disabled } from './FavouritesButton.stories.js'
 
 describe('Component > FavouritesButton', function () {
-  let wrapper
-  const mockTheme = {
-    global: {
-      colors: {
-        statusColors: {
-          error: 'status-error'
-        }
-      }
-    }
-  }
-  before(function () {
-    wrapper = shallow(<FavouritesButton theme={mockTheme} />)
-  })
-
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
-  })
-
-  it('should display an empty icon', function () {
-    const button = wrapper.find(MetaToolsButton)
-    const { icon } = button.props()
-    expect(icon).to.deep.equal(<HeartIcon color='dark-5' fill='none' size='15px' />)
-  })
-
-  it('should not be checked', function () {
-    const checked = wrapper.prop('aria-checked')
-    expect(checked).to.be.false()
-  })
-
-  describe('on click', function () {
-    let onClickStub
+  const user = userEvent.setup()
 
-    before(function () {
-      onClickStub = sinon.stub()
-      wrapper = shallow(<FavouritesButton theme={mockTheme} checked={false} onClick={onClickStub} />)
+  describe('when not favourited', function () {
+    const NotFavouritedStory = composeStory(NotFavourited, Meta)
+      
+    beforeEach(function () {
+      render(<NotFavouritedStory />)
     })
 
-    afterEach(function () {
-      onClickStub.resetHistory()
+    it('should not show a favourited icon', function () {
```

Comment:
Small thing but these tests check the button state and label. They don't check whether the icon is filled or not. Not a blocking issue.